internal/commands/whereisstreamer/whereisstreamer.go

```diff
@@ -0,0 +1,100 @@
+package whereisstreamer
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	
+	"github.com/dghubble/go-twitter"
+	"github.com/dghubble/oauth1"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/pajbot/basecommand"
+	"github.com/pajbot/pajbot2-discord/internal/config"
+	"github.com/pajbot/pajbot2-discord/pkg"
+	"github.com/pajbot/pajbot2-discord/pkg/commands"
+)
+
+func init() {
+	commands.Register([]string{"$whereisclown"}, New())
```

Comment:
added whereisclown for forsen despite being whereisstreamer. Maybe we want to make it generic or make it configurable?